app/models/hxl_stats_view.rb

```diff
@@ -0,0 +1,109 @@
+HXLSTATS_COLUMN_HEADERS =
```

Comment:
Any reason line 1 - 66 are outside of the class? Let's pop them back in. The constants (in this case `HXLSTATS_COLUMN_HEADERS` and `HXL_STATS_TAGS`) usually would live right at the top of the class (but inside it), and the methods `get_counts_by_gender`  and `get_counts_by_age` can be private methods within the class. I.e. stick them under the keyword `private`.